sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -520,18 +520,21 @@ Preconditions:
 
 Parameters:
 
-* `sender` - Node which will be a dependency of `receiver`.
+* `src` - Node which will be a dependency of `dest`.
 
-* `receiver` - Node which will be dependent on `sender`.
+* `dest` - Node which will be dependent on `src`.
 
 Exceptions:
 
 * Throws synchronously with error code `invalid` if a queue is recording
   commands to the graph.
 
-* Throws synchronously with error code `invalid` if `sender` or `receiver`
+* Throws synchronously with error code `invalid` if `src` or `dest`
   are not valid nodes created from the graph.
 
+* Throws synchronously with error code `invalid` if `src` and `dest`
+  are the same node.
+
```

Comment:
```suggestion
* Throws synchronously with error code `invalid` if `src` and `dest`
   are not assigned to current graph object.
```

Reply:
I've incorporated this wording into the second error condition, since I think they are trying to define the same error usage